src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,170 @@
+---
+title: APM Glossary
+sidebar_order: 5
+---
+
+### Hierarchy
+
+Each span can be connected to other spans using a parent-child hierarchy. 
+
+For example, a span called `updateJiraTask` would have child spans like `Check permissions`, `Update Task`, `sql.update`, `send.webhooks`.
+
+Each sub-task would have an id in the attribute `parent_span_id` id of the “updateTask” span.
+
+### Properties
+
+Each span has a unique identifier kept in the `span_id` attribute.
+
+Start time and end time are stored in the `start_timestamp` and `timestamp`. End time is captured in the attribute timestamp and not the end_timestamp for now because of compatibility reasons.
+
+The name of the operation is stored in the “op” parameter. Examples are `http`, `sql.query`, `redis.command`.
+
+Additional data about the operation can be stored in the `data` and `tags` attributes.
+
+### Span
+
+The **span** is the primary building block of a distributed trace, representing an individual unit of work done in a distributed system.
+
+It can be the processing of an HTTP request by a server, the querying of a database, or an HTTP call to an external service.
+
+An example of a span that describes a call to an external service:
```

Comment:
I think we might be better off having an example that connects to the previous text about `createJiraTask`.

Reply:
Okay. :) Let me know the best place to get that code example.